docs/docs/databases/bigquery/getting-started/docker.mdx

```diff
@@ -87,6 +87,9 @@ fill in the necessary details:
 <Thumbnail src='/img/databases/bigquery/bigquery-add-service-account_console_2.12.1.png' alt='Connect
 existing BigQuery database in Hasura Cloud' width='900px' />
 
+<Thumbnail src='/img/databases/bigquery/connect-big-query-db_console_2.10.1.png' alt='Connect
```

Comment:
For this image, there is a horizontal scrollbar left in the crop of the page. Please crop this out. Also, if possible, the name of the environment variable here is different to that which is displayed in the image just before it. Which isn't ideal. eg:

<img width="936" alt="Screenshot 2022-10-18 at 14 19 08" src="https://user-images.githubusercontent.com/94021366/196440940-7b2fff9c-2bc2-460d-8173-4a24d4991dae.png">